infrastructure/lib/atat-iam-stack.ts

```diff
@@ -113,6 +99,99 @@ export class AtatIamStack extends cdk.Stack {
           actions: ["organizations:*"],
           resources: ["*"],
         }),
+        new iam.PolicyStatement({
+          sid: "ForbidInsecureTraffic",
+          effect: iam.Effect.DENY,
+          actions: ["*"],
+          resources: ["*"],
+          conditions: {
+            Bool: { "aws:SecureTransport": "false" },
+          },
+        }),
+      ],
+    });
+
+    const cloudformationManagedPolicy = new iam.ManagedPolicy(this, "CloudFormationPolicies", {
+      statements: [
+        new iam.PolicyStatement({
+          sid: "ForbidCertainActions",
+          effect: iam.Effect.DENY,
+          actions: [
+            "iam:*User*",
+            "iam:CreateAccessKey",
+            "iam:*LoginProfile*",
+            "iam:*MFA*",
+            "iam:*AccountAlias",
+            "iam:*SSHPublicKey",
+            "iam:*ServerCertificate",
+            "iam:*ServiceSpecificCredential",
+            "iam:Generate*",
+            "iam:GetAccountAuthorizationDetails",
+            "iam:GetCredentialReport",
+            "logs:*LogEvents",
+            "logs:*QueryResults",
+            "logs:*LogRecord",
+            "logs:*LogGroupFields",
+            "rds-data:*",
+            "rds-db:*",
+            "rds:CrossRegionCommunication",
+            "rds:Download*DbLogFile*",
+            "s3:BypassGovernanceRetention",
+            "secretsmanager:CancelRotateSecret",
+            "sqs:*Message",
+          ],
+          resources: ["*"],
+        }),
+        new iam.PolicyStatement({
+          sid: "DeprecatedDynamoDbAllows",
+          effect: iam.Effect.ALLOW,
+          actions: ["dynamodb:*"],
+          resources: ["*"],
+        }),
+        new iam.PolicyStatement({
+          sid: "AllowManagingResources",
+          effect: iam.Effect.ALLOW,
+          actions: [
+            "apigateway:*",
+            "cloudformation:*",
+            "cognito-idp:AddCustomAttributes",
+            "cognito-idp:Create*",
+            "cognito-idp:DeleteGroup",
+            "cognito-idp:DeleteIdentityProvider",
+            "cognito-idp:DeleteResourceServer",
+            "cognito-idp:Describe*",
+            "cognito-idp:GetUICustomization",
+            "cognito-idp:SetUICustomization",
+            "cognito-idp:SetUserPoolMfaConfig",
+            "cognito-idp:*TagResource",
+            "cognito-idp:UpdateGroup",
+            "cognito-idp:UpdateIdentityProvider",
+            "cognito-idp:UpdateResourceServer",
+            "cognito-idp:UpdateUserAttributes",
+            "cognito-idp:UpdateUserPool",
+            "cognito-idp:UpdateUserPoolClient",
+            "cognito-idp:UpdateUserPoolDomain",
+            "ec2:*",
+            "iam:*",
```

Comment:
This is more of a sanity check and make sure I understand. But here all of the `iam` permissions (and other services done the same way) will be allowed excepted for the ones denied above.

Reply:
That's correct! Yep!

Reply:
Or like `"cognito-idp:Create*"` allows anything that starts with `Create` from https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazoncognitouserpools.html